### PR TITLE
add json profile and add session context and remove olap limit

### PR DESCRIPTION
**Pull request by miswujian**
add json profile and add session context and remove olap limit

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java [hunk 1]
```diff
@@ -175,6 +176,8 @@ public class SessionVariable implements Serializable, Writable {
 
     public static final String ENABLE_PARALLEL_OUTFILE = "enable_parallel_outfile";
 
+    public static final String ENABLE_LATERAL_VIEW = "enable_lateral_view";
```
Comment (morningman) [inline fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java:179]: Remove this

**Review by morningman — APPROVED** [review]
LGTM

**Review by morningman — DISMISSED** [review]
LGTM

**Comment by morrySnow** [comment]
This PR includes too many commit that already merged into apache/master, is it really ok?

**Comment by morrySnow** [comment]
duplicated with a merged PR: #11279

**Pull request closed without merging**